### ISSUE TITLE
kvexpress fails to build

### ISSUE THREAD

**Issue by isavcic** [DataDog/kvexpress#100]
```
# docker run --rm -v "$PWD":/go/bin/ golang:1.8 go get -u github.com/DataDog/kvexpress
# github.com/DataDog/kvexpress/commands
src/github.com/DataDog/kvexpress/commands/datadog.go:229: cannot use title (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:229: cannot use value (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:229: cannot use "error" (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:242: cannot use title (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:242: cannot use value (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:242: cannot use "error" (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:255: cannot use title (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:255: cannot use value (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:255: cannot use "info" (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:270: cannot use title (type string) as type *string in field value
src/github.com/DataDog/kvexpress/commands/datadog.go:270: too many errors
```

**Comment by darron** [comment]
Looks like the underlying library has been modified a bit:

https://github.com/zorkian/go-datadog-api

If it went back to the v1 version of the library that should fix it.

**Comment by isavcic** [comment]
Indeed, thanks!

By replacing `"github.com/zorkian/go-datadog-api"` with `"gopkg.in/zorkian/go-datadog-api.v1"` in `commands/*.go` files I got it to build successfully.

**Comment by darron** [comment]
Closed by #101
